clang/lib/Sema/HLSLBuiltinTypeDeclBuilder.cpp

```diff
@@ -668,6 +668,26 @@ BuiltinTypeDeclBuilder::addHandleConstructorFromBinding() {
       .finalize();
 }
 
+BuiltinTypeDeclBuilder &
+BuiltinTypeDeclBuilder::addHandleConstructorFromImplicitBinding() {
+  if (Record->isCompleteDefinition())
```

Comment:
Could you explain why this is needed?

Reply:
If I recall correctly, it has something to do with precompiled headers. But now when I tried to remove to see what fails, all of the test pass! So maybe it is not needed anymore?

I would prefer to keep it in for now for consistency with the other constructor and method building functions. I will review all places where this is used and remove them all at once if it is really no longer needed.